.github/workflows/deploy-workflow.yml

```diff
@@ -0,0 +1,91 @@
+name: GCP Deploy
+on: push
+#on:
+#  push:
+#    master
+
+jobs:
+  deploy-app-engine:
+    name: App Engine
+    runs-on: ubuntu-latest
+    env:
+      working-drectory: ./autoscheduler
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Fill in Postgres credentials
+        run: bash fill_credentials.sh ${{ secrets.POSTGRES_USER }} ${{ secrets.POSTGRESS_PASS }}
+
+      - name: Initialize Google Cloud SDK
+        uses: zxyle/publish-gae-action@master
+        with:
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          project_id: ${{ secrets.PROJECT_ID }}
+          gae_config_path: ./autoscheduler/app.yaml
+
+      - name: Initialize GCP Credentials
+        run: |
+          # This client-secret.json is converted by GCP_SA_KEY
+          gcloud auth activate-service-account ${{ secrets.GCP_SA_EMAIL }} --key-file=client-secret.json
+          gcloud config set project ${{ secrets.PROJECT_ID }}

```

Comment:
yeah I think it's fine leaving it like that, just since there's really no difference.

In regards to the warning I have honestly no idea why it would be saying that as I know that service account has permissions for it, so I guess we can just ignore it for now